CMakeLists.txt

```diff
@@ -317,6 +318,10 @@ if(MLIO_INCLUDE_PYTHON_EXTENSION)
     include(PythonExtension)
 endif()
 
+if(MLIO_INCLUDE_OPENCV)
+    find_package(opencv 4.2.0 REQUIRED core imgproc imgcodecs)
```

Comment:
Do we really need the most recent version of opencv?

Reply:
what should be the criteria here? As a newly added feature, I wanted to keep it aligned with most recent OpenCV version, as there is no risk of backwards incompatible changes.

Reply:
Most likely not all of our users will be using the most recent version of OpenCV. We should ideally support the earliest version that supports all the features we rely on. You might check TensorFlow, MXNet, etc. to see which version of OpenCV they are depending on.